CSS/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  max-width: 960px;
+  margin: 0 auto;
+  color: white;
+  background-color: black;
+}
+
+header {
```

Comment:
Overall nice work here. If you're using the same property/value in `header` and `footer`, just assign that rule to a class and put the class on both HTML elements.